content/downloads/_index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Downloads
+menu:
+  main:
+    weight: 50
+---
+
+<!-- {{% blocks/lead color="primary"%}}
+# Resources
+Check out the resources below for more information on using SUSHI.
+{{% /blocks/lead %}} -->
```

Comment:
What's this commented out code for?

Reply:
I goofed, the page was more involved at first, but I decided to just keep it simple and commented that out. Removed in 6424f06.